### PR TITLE
Lab jp

### PR DESCRIPTION
**Pull request by Firefly4266**
Test working and fully functional with a some guidance and direction from Hollis.

### CODE REVIEW

**Comment by Firefly4266** [comment]
Added cow...

File: lib/cowsay.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = function cowsay() {
```
Comment (ankihg) [inline lib/cowsay.js]: this function needs to take in a parameter for what the cow will say and return what the cow will say concatenated with the string representing the cow

File: lib/cowsay.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  const cow = '$the_cow = <<"EOC"'; +
+'          $thoughts   ^__^' +
+'           $thoughts  ($eyes)\\_______' +
+'              (__)\\       )\\/\\' +
+'               $tongue ||----w |' +
+'                  ||     ||';
+  console.log('cowsay', cow);
```
Comment (ankihg) [inline lib/cowsay.js]: it doesn't necessarily need to print it out

File: test/cowsay_test.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const expect = require('chai').expect;
+const cowsay =require('../lib/cowsay.js');
+
+describe('cowsay function', function() {
+  it('This is a trip', function() {
+    const Expected = 'cowsay';
+    expect(cowsay('This is a trip')).to.eql(Expected);
```
Comment (ankihg) [inline test/cowsay_test.js]: what can we expect once the cowsay function returns the inputted string concatenated with the string representing the cow

File: lib/cowsay.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+module.exports = function cowsay() {
+
+  const cow = '$the_cow = <<"EOC"'; +
```
Comment (ankihg) [inline lib/cowsay.js]: the semicolon means end of expression so if you want the string to continue past the line maybe remove it

**Pull request closed without merging**